### PR TITLE
Resolve asyncio issues for 3.13

### PR DESCRIPTION
**Pull request by max-muoto**
Account for new `asyncio` changes from 3.13:

- `abort_clients` and `close_client` methods on `AbstractServer`.
- New `keep_alive` param for `AbstractEventLoop.create_server`.
- Minor helper tweaks.
- Add new `warnings` argument for `asyncio.Streams.StreamWriter.__del__`
- Add new `QueueShutDown` exception and `shutdown` method in `ayncio.queues`.
- Add `EventLoop` to `unix_events.pyi` and `windows_events.pyi`.

### CODE REVIEW

File: stdlib/asyncio/base_events.pyi [hunk 1]
```diff
@@ -237,6 +243,7 @@ class BaseEventLoop(AbstractEventLoop):
             ssl: _SSLContext = None,
             reuse_address: bool | None = None,
             reuse_port: bool | None = None,
+            keep_alive: bool | None = None,
```
Comment (srittau) [inline stdlib/asyncio/base_events.pyi:246]: Weird that the default is `None`, not `False`, but it is what it is ...

File: stdlib/asyncio/queues.pyi [hunk 2]
```diff
@@ -10,11 +10,16 @@ if sys.version_info >= (3, 10):
...
     _LoopBoundMixin = object
 
-__all__ = ("Queue", "PriorityQueue", "LifoQueue", "QueueFull", "QueueEmpty")
-
 class QueueEmpty(Exception): ...
 class QueueFull(Exception): ...
 
+if sys.version_info >= (3, 13):
+    class QueueShutDown(Exception): ...
+    __all__ = ("Queue", "PriorityQueue", "LifoQueue", "QueueFull", "QueueEmpty", "QueueShutDown")
```
Comment (srittau) [inline stdlib/asyncio/queues.pyi:18]: `__all__` should stay at the top of the file (below the imports), even if it means repeating the version_info branch.

File: stdlib/asyncio/unix_events.pyi [hunk 3]
```diff
@@ -59,7 +62,19 @@ if sys.version_info < (3, 14):
 
 if sys.platform != "win32":
     if sys.version_info >= (3, 14):
-        __all__ = ("SelectorEventLoop", "DefaultEventLoopPolicy")
+        __all__ = ("SelectorEventLoop", "DefaultEventLoopPolicy", "EventLoop")
```
Comment (srittau) [inline stdlib/asyncio/unix_events.pyi:65]: I have no idea why `__all__` is down here, but you could also optionally move it to the top of the file.

**Review by srittau — CHANGES_REQUESTED** [review]
Thanks, just one small thing (and an optional small thing) below.

File: stdlib/asyncio/unix_events.pyi (hunk 3 above)
Comment (max-muoto) [reply stdlib/asyncio/unix_events.pyi]: Addressed: https://github.com/python/typeshed/pull/12327/commits/bee3ed25dd11701b855c0f314936097f01b2869c

File: stdlib/asyncio/queues.pyi (hunk 2 above)
Comment (max-muoto) [reply stdlib/asyncio/queues.pyi:17]: Addressed: https://github.com/python/typeshed/pull/12327/commits/bee3ed25dd11701b855c0f314936097f01b2869c

**Review by srittau — APPROVED** [review]

**Pull request merged**